WTWSMS/common/dynasties/00_dynasties.txt

```diff
@@ -32381,16 +32387,19 @@
 	culture="bedouin_arabic"
 	religion="sunni"
 }
-1044033=
+
+1051146=
 {
-	name="Agilolfing"
-	culture="german"
-	religion="catholic"
-	coat_of_arms=
-	{
-		data=
-		{
-0 8 0 6 0 0 0 			}
-		religion="catholic"
-	}
```

Comment:
Here you have removed an existant dynasty, why?

Reply:
Number ordering, I'll add it back

Reply:
Solved